src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import debounce from 'lodash/debounce';
```

Comment:
Fix it
<img width="814" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/8088970c-6b6b-4c20-98a9-e63fa0650e27">